velox/substrait/VeloxToSubstraitExpr.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/core/PlanNode.h"
+
+#include "velox/substrait/VeloxToSubstraitFunc.h"
+#include "velox/substrait/VeloxToSubstraitType.h"
+#include "velox/substrait/proto/substrait/algebra.pb.h"
+
+using namespace facebook::velox::core;
+
+namespace facebook::velox::substrait {
+
+class VeloxToSubstraitExprConvertor {
+ public:
+  // Convert Velox Expression to Substrait Expression.
+  void toSubstraitExpr(
+      const std::shared_ptr<const ITypedExpr>& vExpr,
+      RowTypePtr vPreNodeOutPut,
+      ::substrait::Expression* sExpr);
+
+  // Convert Velox Constant Expression to Substrait
+  // Literal Expression.
+  void toSubstraitLiteral(
+      const velox::variant& vConstExpr,
```

Comment:
I was expecting this function to take the actual const expression, not a variant.

Reply:
fix this.